apps/src/templates/sectionAssessments/StudentsMCSummaryContainer.jsx

```diff
@@ -0,0 +1,29 @@
+import React, {Component, PropTypes} from 'react';
+import StudentsAssessmentsMCTable, {studentOverviewDataPropType} from './StudentsAssessmentsMCTable';
+import {
+  getStudentsMCSummaryForCurrentAssessment,
+} from './sectionAssessmentsRedux';
+import {connect} from 'react-redux';
+
+class StudentsMCSummaryContainer extends Component {
+  static propTypes = {
+    studentOverviewData: PropTypes.arrayOf(studentOverviewDataPropType),
+  };
+
+  render() {
+    return (
+      <div>
+        <h1>Students MC Summary Container </h1>
```

Comment:
is this header intentional?

Reply:
This is a place holder for the actual header for the table.  Comments will be added for clarification.